presto-sqlserver/src/main/java/com/facebook/presto/plugin/sqlserver/SqlServerClient.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.sqlserver;
+
+import com.facebook.presto.plugin.jdbc.BaseJdbcClient;
+import com.facebook.presto.plugin.jdbc.BaseJdbcConfig;
+import com.facebook.presto.plugin.jdbc.JdbcConnectorId;
+import com.microsoft.sqlserver.jdbc.SQLServerDriver;
+
+import javax.inject.Inject;
+
+import java.sql.SQLException;
+
+public class SqlServerClient
+        extends BaseJdbcClient
+{
+    @Inject
+    public SqlServerClient(JdbcConnectorId connectorId, BaseJdbcConfig config)
+            throws SQLException
+    {
+        super(connectorId, config, "\"", new SQLServerDriver());
+    }
+}
```

Comment:
SqlServer supports transactional DDLs. Why don't you use it instead of relying on default implementation of `BaseJdbcClient` providing non-ACID compliant DDLs?